APIServer/api_endpoints.py

```diff
@@ -237,7 +263,6 @@ def get(self):
         return agent.to_json()
 
 
-@api.route('/registry/get/<int:exec_key>')
 class GetRegistry(Resource):
```

Comment:
Is this for fetching registry for a particular exec_key?
Can I move this to `Registry(Resource)` instead?

Reply:
I think it is nonsense. I did not yank it out because I did not have time to see if anyone is using / testing it.

Reply:
In other words, I don't think it makes sense to query the registry for a particular exec key. We only want to look at it because perhaps for some reason we are holding the wrong exec key.

Reply:
Oh I see.
I tried to see when was this committed but seems like it was part of the initial commit for the new repo.